src/url-handling.js

```diff
@@ -188,7 +188,7 @@ export function stateFromLocation(location: Location): UrlState {
 
   const pathParts = pathname.split('/').filter(d => d);
   const dataSource = getDataSourceFromPathParts(pathParts);
-  const selectedThread = query.thread !== undefined ? +query.thread : 0;
+  const selectedThread = query.thread !== undefined ? +query.thread : null;
```

Comment:
I changed it so that it doesn't default to 0 if no thread is selected. This will indicate to the VIEW_PROFILE action that it needs to decide how to present the threads.